infrastructure/main.tf

```diff
@@ -12,6 +12,13 @@ module "eks" {
   source = "./module/eks"
 
   node_subnets_id = module.vpc.private_subnets
-  control_subnets_id = module.vpc.private_subnets
   vpc_id = module.vpc.id
 }
+
+module "psql" {
+  source = "./module/database"
+
+  eks_cluster_name = module.eks.eks_cluster_name
+
+  password = var.db_password // secret db password is passed as evn
```

Comment:
This is not needed as we don't want to hardcore pass in the github files.
Values should be empty and you will populate them manually inside secret manager.